scripts/watch-all.js

```diff
@@ -54,10 +54,12 @@ function whenQuiet(childProcess, timeout = 1000) {
 
         // wait until stdout is quiet for a while before resolving this promise.
         const resolveDebounced = debounce(() => {
+            childProcess.stderr.on('data', () => resolveDebounced());
```

Comment:
I noticed that when Buildpack or Peregrine failed the initial build, the watch:all script didn't display errors. This fixes that.